examples/ai_examples/vllm/vllm_configuration.yml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: vllm-llama-app
+  name: vllm-llama-svc
+  namespace: workloads
+spec:
+  ports:
+    - port: 8000
+      protocol: TCP
+      targetPort: 8000
+  selector:
+    app: vllm-llama-app
+  type: NodePort
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: vllm-llama-app
+  name: vllm-llama
+  namespace: workloads
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: vllm-llama-app
+  template:
+    metadata:
+      labels:
+        app: vllm-llama-app
+    spec:
+      containers:
+        - image: vault.habana.ai/gaudi-docker/1.17.1/ubuntu22.04/habanalabs/pytorch-installer-2.3.1:latest
+          name: vllm-llama-openai
+          imagePullPolicy: Always
+          workingDir: /root
+          env:
+            - name: HF_HOME
+              value: /storage/huggingface
+            - name: http_proxy
+              value: ""
+            - name: https_proxy
+              value: ""
+            - name: no_proxy
+              value: ""
+            - name: LLM_MODEL
+              value: meta-llama/Meta-Llama-3-70B-Instruct
+            - name: HUGGING_FACE_HUB_TOKEN
+              value: ""
+            - name: HABANA_VISIBLE_DEVICES
+              value: all
+            - name: NUM_HPU
+              value: "4"
+            - name: OMPI_MCA_btl_vader_single_copy_mechanism
+              value: none
+            - name: PT_HPU_ENABLE_LAZY_COLLECTIVES
+              value: "true"
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+              git clone -b v0.4.2-Gaudi-1.16.0 https://github.com/HabanaAI/vllm-fork.git
```

Comment:
please use v0.5.3.post1-Gaudi-1.17.0 branch as it has many performance fixes and goes well with Habana 1.17

Reply:
Addressed changes in internal PR. It will be available in next set of PRs

Reply:
Thank you